etest/docker/image.py

```diff
@@ -19,18 +19,18 @@ def pull(image_name):
     image_id = None
 
     try:
-        image_id = common.CLIENT.inspect_image(image_name)["Id"]
+        image_id = common.API_CLIENT.inspect_image(image_name)["Id"]
     except docker.errors.APIError as error:
         if error.response.status_code != 404:
             raise error
 
     repository, tag = image_name.split(":")
 
-    common.CLIENT.pull(repository=repository, tag=tag)
+    if image_id is not None and image_id != common.API_CLIENT.inspect_image(image_name)["Id"]:
```

Comment:
Why does this conditional not include the remove_image as well anymore?

Reply:
I dont remember removing it. It might be a human error when merging.